release-notes/v1_30.md

```diff
@@ -653,15 +653,6 @@ Below are some of the new features:
 
 You can read the full release notes for the extension at [vscode-pull-request-github releases](https://github.com/microsoft/vscode-pull-request-github/releases).
 
-## New Documentation
-
-### Vue.js and Ember.js tutorials
-
-There are two new tutorials showing how to use VS Code with these popular JavaScript frameworks.
-
-* [Using Vue in Visual Studio Code](https://code.visualstudio.com/docs/nodejs/vuejs-tutorial)
```

Comment:
Should the vue one still be kept?

Reply:
It was in an old release note and also the Vue.js tutorial is currently stale (needs to be updated for Volar extension) so I figured I'd just remove the section